strings/en_US/messages.po

```diff
@@ -3531,3 +3531,6 @@ msgstr "Try modifying your search location, or zoom out on the {openLink}map{clo
 
 msgid "Contact Dojo"
 msgstr "Contact Dojo"
+
+msgid "FULL"
```

Comment:
As per https://github.com/CoderDojo/cp-zen-frontend/pull/2#discussion_r126376064 we should just use `Full`